app/controllers/v1/statement_of_cases_controller.rb

```diff
@@ -0,0 +1,46 @@
+module V1
```

Comment:
Is this in the API structure for a reason?

Reply:
Yes - discussed and it may be refactored/renamed in the future due to naming similarities but for now it is where it should be :+1: